.circleci/config.yml

```diff
@@ -423,12 +451,6 @@ jobs:
             #!/bin/bash
 
             GH_LABEL=team-mmi
-            if [ -z "$CIRCLE_PULL_REQUESTS" ]; then
-              echo "Skipping tag check; this is not a PR."
-              echo "false" > ./RUN_MMI_OPTIONAL
```

Comment:
doesn't the fact that we don't assign `./RUN_MMI_OPTIONAL` anymore substantially change the script?